tethys_apps/base/app_base.py

```diff
@@ -104,69 +105,132 @@ def persistent_stores(self):
                 \"""
 
                 stores = (PersistentStore(name='example_db',
-                                          initializer='init_stores:init_example_db',
-                                          spatial=True
+                                          initializer='my_first_app.init_stores.init_example_db',
+                                          spatial=True,
                         ),
                 )
 
                 return stores
         """
         return None
 
-    def dataset_services(self):
+    def custom_settings(self):
+        """
+        Use this method to define custom settings for use in your app.
+
+        Returns:
+          iterable: A list or tuple of ``CustomTethysAppSetting`` objects.
+
+        **Example:**
+
+        ::
+
+            from tethys_sdk.settings import CustomTethysAppSetting
+            def custom_settings(self):
+                \"""
+                Example general_settings method.
+                \"""
+                custom_settings = (
+                    CustomTethysAppSetting(
+                           name='example',
+                           description='dataset service for app to use',
+                           required=True,
+                    ),
+                )
+
+                return custom_settings
+        """
+        return None
+
+    def persistent_store_service_settings(self):
+        """
+        Define this method to define a persistent store service connections for your app.
+
+        Returns:
+          iterable: A list or tuple of ``PersistentStoreServiceSettings`` objects. A persistent store database will be created for each object returned.
+
+        **Example:**
+
+        ::
+
+            from tethys_sdk.settings import PersistentStoreServiceSetting
+
+            def persistent_stores(self):
+                \"""
+                Example persistent_store_service_settings method.
+                \"""
+
+                stores = (
+                    PersistentStoreServiceSetting(
+                        name='example_db',
+                        initializer='init_stores:init_example_db',
```

Comment:
The initializer string in the example is incorrect. Should be 'my_first_app.init_stores.init_example_db'.